public/app/core/services/backend_srv.ts

```diff
@@ -66,6 +66,11 @@ export class BackendSrv {
     var requestIsLocal = options.url.indexOf('/') === 0;
     var firstAttempt = options.retry === 0;
 
+    if (!options.url.match('https?://') && this.contextSrv && this.contextSrv.user && this.contextSrv.user.orgId) {
```

Comment:
Initially I was using `requestIsLocal`, but that is false for several of the calls that worldping-app makes, since it uses urls like `api/...` eg https://github.com/raintank/worldping-app/blob/master/src/components/config/config.js#L62